pkg/security/generators/accessors/accessors.tmpl

```diff
@@ -297,7 +298,129 @@ func (ev *Event) GetFieldMetadata(field eval.Field) (eval.EventType, reflect.Kin
 	return "", reflect.Invalid, "", &eval.ErrFieldNotFound{Field: field}
 }
 
+func (ev *Event) setStringArray(field string, fv *[]string, value interface{}) error {
```

Comment:
Do we really need those in the template ? They could be moved to a real go file no?